code/core/src/core-server/withTelemetry.ts

```diff
@@ -18,11 +25,12 @@ const promptCrashReports = async () => {
     return undefined;
   }
 
-  const { enableCrashReports } = await prompts({
-    type: 'confirm',
-    name: 'enableCrashReports',
-    message: `Would you like to help improve Storybook by sending anonymous crash reports?`,
-    initial: true,
+  const enableCrashReports = await prompt.confirm({
+    message: dedent`
+      Send anonymous crash reports to help improve Storybook?
+      This helps us improve Storybook and fix bugs faster.
```

Comment:
You're mentioning _help improve_ twice without any further details. Seems superfluous.